deployment/service_template_generator.py

```diff
@@ -298,12 +298,13 @@ def _add_alb(self, cd, service_name, config):
         self.template.add_resource(alb)
 
         target_group_name = "TargetGroup" + service_name
+        health_check_path = config['http_interface']['health_check_path'] if config['http_interface']['health_check_path'] else "/elb-check"
```

Comment:
This check fails when an service created by an older version of cloudlift is updated with this version. Checking for the existence of the key before accessing it, solves it.

`health_check_path = config['http_interface']['health_check_path'] if 'health_check_path' in config['http_interface'] else "/elb-check"`

Reply:
Oops. Fixed it. Thanks